{{cookiecutter.github_repository}}/.travis.yml

```diff
@@ -7,7 +7,10 @@ python:
 - '3.5.2'
 
 addons:
-  postgresql: "9.4"
+  postgresql: "9.5"
```

Comment:
Are we also ensuring postgres 9.5 on ubuntu 16?

Reply:
checked, ubuntu has postgres 9.5, so we are good to go with this.

Reply:
ok, I tried it on 2 projects and it is running fine.